src/test/ui/rfc-2565-param-attrs/param-attrs.rs

```diff
@@ -0,0 +1,212 @@
+// compile-pass
+// revisions: bar foo
+
+#![feature(param_attrs)]
+
+extern "C" {
+    fn ffi(
+        #[allow(bar)] a: i32,
```

Comment:
Does this test actually account for the lint check attributes here having any effect?
If not here, then some test should do that.

Reply:
This can be checked by enabling `#![deny(unused_variables)]` for the test and using `#[allow(unused_variables)]` as the fn param attributes, similarly to the `param-attrs-cfg.rs` test.

Reply:
Deferred to https://github.com/rust-lang/rust/issues/61238.

Reply:
Yep, sgtm.